static/src/javascripts/components/raven-js/raven.js

```diff
@@ -1410,7 +1410,14 @@ var Raven = {
      */
     lastEventId: function() {
         return lastEventId;
-    }
+    },
+
+    /*
+     * Has raven been configured?
+     *
+     * @return {boolean}
+     */
+    isSetup: isSetup
```

Comment:
This is a third party script, are we going to allow modifying them? I pity the developer who will manage our dependencies in the future. Another example is here: https://github.com/guardian/frontend/commit/c6202a8926e9da0ff132de77edf56fe40e8a93bb